examples/job/job.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Job
+metadata:
+  name: test-job
+spec:
+  parallelism: 2
+  completions: 5
+  selector:
```

Comment:
Actually this is done at defaulting level, and our examples_test.go calls validation directly, so I need to leave it this here, or we'll fix that test.